frontend/src/views/Results.tsx

```diff
@@ -109,11 +153,105 @@ function GameResultsPage() {
       });
   };
 
+  const isHost = useCallback((user: User | null) => user?.userId === host?.userId, [host]);
+
+  // Get current mouse position.
+  const mouseMoveHandler = useCallback((e: MouseEvent) => {
+    setMousePosition({ x: e.pageX, y: e.pageY });
+  }, [setMousePosition]);
+
+  useEffect(() => {
+    window.onmousemove = mouseMoveHandler;
+  }, [mouseMoveHandler]);
+
+  // Content to display for inviting players (if not enough players on the podium)
+  const inviteContent = () => (
+    <InviteContainer>
+      <InviteText
+        onClick={() => {
+          copy(`https://codejoust.co/play?room=${roomId}`);
+          setCopiedRoomLink(true);
+        }}
+      >
+        Invite
+        <InlineCopyIcon>content_copy</InlineCopyIcon>
+      </InviteText>
+    </InviteContainer>
+  );
+
   return (
     <Content>
+      <CopyIndicatorContainer copied={copiedRoomLink}>
+        <CopyIndicator onClick={() => setCopiedRoomLink(false)}>
+          Link copied!&nbsp;&nbsp;✕
+        </CopyIndicator>
+      </CopyIndicatorContainer>
+      <HoverTooltip
+        visible={hoverVisible}
+        x={mousePosition.x}
+        y={mousePosition.y}
+      >
+        Only the host can restart the room
+      </HoverTooltip>
+      <LargeText>Winners</LargeText>
+      <PodiumContainer>
+        <Podium
+          place={2}
+          player={players[1]}
+          gameStartTime={startTime}
+          inviteContent={inviteContent()}
+          loading={loading}
+        />
+        <Podium
+          place={1}
+          player={players[0]}
+          gameStartTime={startTime}
+          inviteContent={inviteContent()}
+          loading={loading}
+        />
+        <Podium
+          place={3}
+          player={players[2]}
+          gameStartTime={startTime}
+          inviteContent={inviteContent()}
+          loading={loading}
+        />
+      </PodiumContainer>
+
+      <div>
+        <HoverContainer>
```

Comment:
I noticed a bug with the hover element on this particular page: [screencast of inaccurate hover tooltip](https://www.loom.com/share/f8c1f850424b46abb85c90a57d7eb6a9). I think this issue doesn't exist on the lobby page because when I try to hover over the "Start Game" button when it's loading, it doesn't give that hover tooltip.

I'm ok with this being fixed in this PR or the next since it's a small bug and the user can remove it by just refreshing the page again.

Reply:
Good catch! Added a loading check to not enable the hover component when initially loading